modules/identity-provider-add.adoc

```diff
@@ -12,6 +12,17 @@
 // * authentication/identity_providers/configuring-google-identity-provider.adoc
 // * authentication/identity_providers/configuring-oidc-identity-provider.adoc
 
+// OIDC, GitHub, and Google IDPs do not support username/password login commands
+ifeval::["{context}" == "configuring-oidc-identity-provider"]
+:no-username-password-login:
+endif::[]
```

Comment:
This does not need to be true when (if) we've fixed https://bugzilla.redhat.com/show_bug.cgi?id=1727983

Reply:
If this does get resolved, just let me know and we can remove it for this case.